application/extensions/bulma-richtext/src/main/webapp/libs/bulma-richtext/web-resources/components/richtext/ui/ColorDialog.tsx

```diff
@@ -110,14 +112,25 @@ const ColorDialog = ({ configuration, state, action }) => {
 
     useEffect(() => {
         createColorsList(colors);
-        
         injectGlobal`
             .ProseMirror {
               ${createGlobalCSS(COLORS)}
             }
         `;
     }, []);
 
+    const filterColor = (colorClassName: string) => {
+        const filteredColor = COLORS.filter(color => color.colorClassName === colorClassName);
+        if (filteredColor.length) {
+            return filteredColor[0].value;
```

Comment:
@martyna-ds we are returning here the first color, given that we do not need to use above the `.filter` as it iterates over all array elements. Here it would be better to use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find isntead of filter as `.find` will return the first found value which is exacly what the `filteredColor[0].value` is doing

so:
```
const color = COLORS.find(color => color.colorClassName === colorClassName);

return color ? color[0].value : RTE_DEFAULT_BUTTON_COLOR;
```